README.md

```diff
@@ -14,144 +14,231 @@
 Cognite Replicator is a Python package for replicating data across Cognite Data Fusion (CDF) projects. This package is
 built on top of the Cognite Python SDK.
 
-Copyright 2019 Cognite AS
+Copyright 2023 Cognite AS
 
 ## Prerequisites
 In order to start using the Replicator, you need:
-* Python3 (>= 3.6)
-* Two API keys, one for your source tenant and one for your destination tenant. Never include the API key directly in the code or upload the key to github. Instead, set the API key as an environment variable.
-
-This is how you set the API key as an environment variable on Mac OS and Linux:
+- Python3 (>= 3.6)
```

Comment:
Should we also mention here that "this component is Community content and not officially supported by Cognite. Bugs and changes will be fixed on a best effort basis" ?